### PR TITLE
doc: modifed the reaction-search documentation

### PR DESCRIPTION
**Pull request by KrittapasJ**
- Added images
- Modified documentation
- Remove incorrect information

### CODE REVIEW

**Comment by kjappelbaum** [comment]
are there some bugs you found in the view?

**Comment by KrittapasJ** [comment]
I don't think so.

Set



Sent from my Galaxy



-------- Original message --------
From: Kevin Jablonka ***@***.***>
Date: 5/2/21 11:32 (GMT+01:00)
To: cheminfo/c6h6-documentation ***@***.***>
Cc: Jantarug Krittapas ***@***.***>, Author ***@***.***>
Subject: Re: [cheminfo/c6h6-documentation] doc: modifed the reaction-search documentation (#21)


are there some bugs you found in the view?

File: src/book/eln/eln-search/README.md [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
-- similarity: search for samples with a similar structure.
+Next, choose between the two search modes at the search mode dropdown box on the bottom in left module:
 
-The center module displays the list of matching structures. Select the one you want.
-The right module displays the list of samples associated with the selected structure. Double clicking a row will take you back to the main tab, with the corresponding sample selected in the List of selected samples.
+![Search mode](images/searchmode.png)
 
-Double clicking a row in the center module will take you directly to the first matching sample.
+- `substructure` : search for samples containing the given substructure
+- `similarity` : search for samples with a similar structure.
```
Comment (kjappelbaum) [inline src/book/eln/eln-search/README.md:20]: can you define similarity

File: src/book/eln/eln-search/README.md [hunk 2]
```diff
@@ -1,23 +1,44 @@
...
+
+Then, draw a structure in the left module.
+
+![Left module](images/leftmodule.png)
+
+The list of matching structures would be displayed at the center module. 
+You can also know which of the interested structure is reagent or product by checking the right two columns.
```
Comment (kjappelbaum) [inline src/book/eln/eln-search/README.md]: seems here is a typo "interested structure"

**Review by kjappelbaum — CHANGES_REQUESTED** [review]
besides the comments lgtm

**Comment by KrittapasJ** [comment]
Hi Kevin,

I change “ interested structure" to be "structure of interest” already. Is it better?
According to definition of similarity, I try to study this mode but I am actually not sure about definition of it. Could you please explain more about similarity mode?
Thank you in advance

Sincerely,
Set

On 11 May 2564 BE, at 21:17, Kevin Jablonka ***@***.******@***.***>> wrote:



@kjappelbaum requested changes on this pull request.

besides the comments lgtm

________________________________

In src/book/eln/eln-search/README.md<https://github.com/cheminfo/c6h6-documentation/pull/21#discussion_r630454465>:

>
-Double clicking a row in the center module will take you directly to the first matching sample.
+- `substructure` : search for samples containing the given substructure
+- `similarity` : search for samples with a similar structure.


can you define similarity

________________________________

In src/book/eln/eln-search/README.md<https://github.com/cheminfo/c6h6-documentation/pull/21#discussion_r630454813>:

>
-Double clicking a row in the center module will take you directly to the first matching sample.
+- `substructure` : search for samples containing the given substructure
+- `similarity` : search for samples with a similar structure.
+
+Then, draw a structure in the left module.
+
+Left module
+
+The list of matching structures would be displayed at the center module.
+You can also know which of the interested structure is reagent or product by checking the right two columns.


seems here is a typo "interested structure"

**Comment by KrittapasJ** [comment]
Hi Kevin,

During last class, I was asking about the definition of similarity for reaction search, Luc found that it might be a bug of this mode. So, what should I do next?
Thank you in advance.

Sincerely

Set

On 11 May 2564 BE, at 21:17, Kevin Jablonka ***@***.******@***.***>> wrote:



@kjappelbaum requested changes on this pull request.

besides the comments lgtm

________________________________

In src/book/eln/eln-search/README.md<https://github.com/cheminfo/c6h6-documentation/pull/21#discussion_r630454465>:

>
-Double clicking a row in the center module will take you directly to the first matching sample.
+- `substructure` : search for samples containing the given substructure
+- `similarity` : search for samples with a similar structure.


can you define similarity

________________________________

In src/book/eln/eln-search/README.md<https://github.com/cheminfo/c6h6-documentation/pull/21#discussion_r630454813>:

>
-Double clicking a row in the center module will take you directly to the first matching sample.
+- `substructure` : search for samples containing the given substructure
+- `similarity` : search for samples with a similar structure.
+
+Then, draw a structure in the left module.
+
+Left module
+
+The list of matching structures would be displayed at the center module.
+You can also know which of the interested structure is reagent or product by checking the right two columns.


seems here is a typo "interested structure"

**Comment by kjappelbaum** [comment]
don't worry about this bug for the course.

**Review by kjappelbaum — APPROVED** [review]

**Pull request merged**